tests/spec/fs.writeFile-readFile.spec.js

```diff
@@ -99,3 +99,118 @@ describe('fs.writeFile, fs.readFile', function() {
     });
   });
 });
+
+
+
+
+
+describe('fs.promises.writeFile, fs.promises.readFile', function() {
+  beforeEach(util.setup);
+  afterEach(util.cleanup);
+
+  it('should be a function', function() {
+    var fs = util.fs();
```

Comment:
This will work, but it's more common for people to grab the `promises` object and cache it here, like so:

```js
var fsPromises = util.fs().promises;
...
fsPromises.writeFile()...
```

This way you don't have to do `fs.promises` all the time below.  This same change could happen throughout your code.